src/Microsoft.OData.Edm/Validation/InterfaceValidator.cs

```diff
@@ -314,6 +314,22 @@ private IEnumerable<EdmError> ValidateStructure(object item)
                 }
             }
 
+            // Out-of-line annotations are found through the model visitor, but inline annotations
+            // are not found through normal traversal, so add in-line annotations here.
+            IEdmVocabularyAnnotatable annotatable = item as IEdmVocabularyAnnotatable;
+            if (this.model != null && annotatable != null)
+            {
+                foreach (IEdmVocabularyAnnotation annotation in annotatable.VocabularyAnnotations(this.model))
+                {
+                    // Inline annotations have a null target.
+                    if (annotation.Target == null)

```

Comment:
>annotation.Target == null [](start = 24, length = 25)

So, do we use "annotation.Target == null" to distinguish inline from outOfLine?

Reply:
Yes.

---
In reply to: [174975029](https://github.com/OData/odata.net/pull/1109#discussion_r174975029) [](ancestors = 174975029)